Lib/Protocols/IdSSLOpenSSLHeaders.pas

```diff
@@ -18922,6 +18922,26 @@ function RAND_status() : integer;
 function RAND_event(iMsg : UINT; wp : wparam; lp : lparam) : integer;
 procedure RAND_screen();
 {$ENDIF}
+//
+// Zdravko Gabrovski - Global openSSL handlers implementation
+//
+procedure InitializeFuncPointers;
+
+{$IFNDEF STATICLOAD_OPENSSL}
+function GlobalIdSSL : TIdLibHandle;
+function GlobalIdCrypto : TIdLibHandle;
+procedure SetGlobalIdSSL ( fGlobalIdSSL : TIdLibHandle );
+procedure SetGlobalIdCrypto ( fGlobalIdCrypto : TIdLibHandle );
+var
+  hGlobalIdSSL    : TIdLibHandle = IdNilHandle;
+  hGlobalIdCrypto : TIdLibHandle = IdNilHandle;
+  csGlobalIDSSL   : TIdCriticalSection;
+
+  hIdSSL    : TIdLibHandle = IdNilHandle;
+  hIdCrypto : TIdLibHandle = IdNilHandle;
+
+{$ENDIF}
+

```

Comment:
What is the point of declaring getter/setter functions if you are also making the variables themselves public as well? The variables should not be in the `interface`, only the `implementation`.